server/users/methods.js

```diff
@@ -169,3 +176,29 @@ Meteor.methods({
     return true;
   }
 });
+
+/**
+ * get users support link
+ * @function
+ * @name users.getSupportLinks
+ * @param { String } - userId
+ * @return {Object}
+ */
+
+Meteor.methods({
+  "users.getSupportLink"(userId) {
+    check(userId, String);
+
+    const user = Meteor.users.findOne(
+      { _id: userId },
+      { fields: { "profile.support_links": 1 } }
+    );
+    if (user && user.profile && user.profile.support_links) {
+      console.log("here");
```

Comment:
Do we need this debug log?

Reply:
@sergeant-q nice catch. no we don't need that. I've removed it. :)